### ISSUE TITLE
FL87 false negative?

### ISSUE THREAD

**Issue by knocte** [fsprojects/FSharpLint#801]
```
% sudo dotnet tool install --global dotnet-fsharplint 
Skipping NuGet package signature verification.
You can invoke the tool using the following command: dotnet-fsharplint
Tool 'dotnet-fsharplint' (version '0.26.8') was successfully installed.
% dotnet fsharplint lint sprintf String.Empty
========== Linting <inline source> ==========
========== Finished: 0 warnings ==========
========== Summary: 0 warnings ==========
```
